{"name":"test4","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"null","trace":"java.lang.AssertionError: null\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat org.testng.Assert.fail(Assert.java:106)\r\n\tat org.testng.rerunningfailedtests.RerunFailedTestClass2.test4(RerunFailedTestClass2.java:11)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.util.ArrayList.forEach(Unknown Source)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:764)\r\n\tat org.testng.TestRunner.run(TestRunner.java:585)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:286)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1140)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1069)\r\n\tat org.testng.TestNG.run(TestNG.java:1037)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1609203798885,"stop":1609203798888,"uuid":"a6ef9eea-f0fc-4de6-8a2a-be826d59d8cc","historyId":"d9808e569b06937416d8ca7c5744b204","fullName":"org.testng.rerunningfailedtests.RerunFailedTestClass2.test4","labels":[{"name":"package","value":"org.testng.rerunningfailedtests.RerunFailedTestClass2"},{"name":"testClass","value":"org.testng.rerunningfailedtests.RerunFailedTestClass2"},{"name":"testMethod","value":"test4"},{"name":"parentSuite","value":"Rerun Failed Tests"},{"name":"suite","value":"Rerunning failed tests"},{"name":"subSuite","value":"org.testng.rerunningfailedtests.RerunFailedTestClass2"},{"name":"host","value":"KARMA"},{"name":"thread","value":"20304@KARMA.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}